symengine/polys/uintpoly_piranha.h

```diff
@@ -70,6 +108,36 @@ class UIntPolyPiranha : public UIntPolyBase<pintpoly, UIntPolyPiranha>
         return poly_.degree();
     }
 
+    // begin() and end() are unordered
+    // rbegin() and rend() are ordered
```

Comment:
I thought it was better to stick to the standard name. I'll make the change, but what's the problem with this, right now?